components/collections/src/codepointtrie/benches/tries/gc_fast.rs

```diff
@@ -3,10 +3,11 @@
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
 use icu_collections::codepointtrie::{CodePointTrieHeader, TrieType};
-use zerovec::{ule::RawBytesULE, ZeroSlice};
+use zerovec::ule::AsULE;
+use zerovec::{zeroslice, ZeroSlice};
 
 #[rustfmt::skip]
-pub static INDEX: &ZeroSlice<u16> = ZeroSlice::<u16>::from_ule_slice(&RawBytesULE::<2>::from_array([
+pub static INDEX: &ZeroSlice<u16> = zeroslice![u16; <u16 as AsULE>::ULE::from_array;
```

Comment:
Because these are `static`s [1], I'd have to either `.clone()` them at the use-site, or turn them into `const`s. Both options should be similarly cheap at the moment (pointer copy for clone, nothing for const), but if I understand `const`s correctly, then they might lead to code size increases if we start using those (rather large) `const`s in more than one place. Are there any plans for that?

[1]:
cannot move out of static item `DATA` [E0507] move occurs because `DATA` has type `ZeroVec<'_, u8>`, which does not implement the `Copy` trait

Reply:
ah, I didn't see that that whole code lives in `benches`, so I suppose it doesn't matter that much for now

Reply:
ah the trick is to make these `const` instead of `static`